apple/swift-docc-render
# Prevent scrolling on `navigation` event if router `meta` field `preventScrolling` is `true`

Bug/issue #, if applicable: rdar://103146273

## Summary
If `preventScrolling` in the `meta` field is set to `true`, then we should not change the scroller location when navigating to a new page.

## Dependencies
rdar://103119754

## Testing
Steps:
1. set `preventScrolling` to `true` for a page (i.e. '/foo')
2. Scroll down a little bit on any page (i.e. '/bar').  ('/foo' should be longer than '/bar')
3. trigger a navigation event to '/foo'
4. Verify that the scroller did not move up/down nor left/right

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
